src/widget/woverview.cpp

```diff
@@ -592,6 +592,10 @@ void WOverview::mousePressEvent(QMouseEvent* e) {
                     m_pCurrentTrack->removeCue(pHoveredCue);
                     return;
                 } else {
+                    // Clear the pickup position display.
+                    // ev could actually be any event type, it's not used anyway.
+                    QEvent* ev = new QEvent(QEvent::HoverLeave);
+                    leaveEvent(ev);
```

Comment:
ev is leaked. This should do the trick:
```suggestion
                    QEvent ev = QEvent(QEvent::HoverLeave);
                    leaveEvent(&ev);
```

Reply:
Thank you.
I definitly have to do some homework wrt pointers.

Reply:
done.

Reply:
when in doubt, never write "new" :).